### DISCUSSION TITLE
Monai Valve landmark detection

### DISCUSSION THREAD

**Discussion by shrutiihegde** [Project-MONAI/MONAI#7307] [Q&A]
<img width="1120" alt="Screenshot 2023-12-09 at 4 04 58 PM" src="https://github.com/Project-MONAI/MONAI/assets/98289103/5ee667e9-7d55-4713-ab22-c36e00445bec">
<img width="1120" alt="Screenshot 2023-12-09 at 4 05 19 PM" src="https://github.com/Project-MONAI/MONAI/assets/98289103/e887c454-d738-4356-8bbb-4b45a40494c5">
<img width="1120" alt="Screenshot 2023-12-09 at 4 05 34 PM" src="https://github.com/Project-MONAI/MONAI/assets/98289103/11e0f7c7-49c2-40b3-a68e-7ceb5245d56b">

I am trying to run the inference step of Monai Valve landmark detection using the model, config file and data from the model zoo GitHub. The error I get is: RuntimeError: Failed to evaluate ConfigExpression: "$__local_refs['evaluator'].run()".

 Any suggestions?

**Comment by KumoLiu** [comment]
HI @shrutiihegde, the root cause is `number of dim don't match in permute`.
It means in `Transpose`, the specified indices isn't match with the data shape.

Hope it helps, thanks!
